### PR TITLE
MultireferenceReadonly and SingleReferenceReadOnly changes added

### PR DESCRIPTION
**Pull request by 4manasa**
Created components to render MultireferenceReadonly and SingleReferenceReadonly

### CODE REVIEW

File: src/components/templates/SingleReferenceReadonly/index.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  }
+
+  getSingleReferenceHtml(): any {
+    const singleRefHtml = html`<semantic-link .pConn=${this.pConn}></semantic-link>`;
+
+    return singleRefHtml;
+  }
+
+  render(){
+    console.log('pConnn singleref', this.pConn);
```
Comment (JEAxero) [inline src/components/templates/SingleReferenceReadonly/index.ts:78]: pConnn has 3 ds in this console.log but I think this should be removed or commented out to remove console clutter.

**Review by JEAxero — CHANGES_REQUESTED** [review]
See comment re: moving a console.log. Also, when I try to test this, I'm still seeing some errors like this:

Uncaught (in promise) Error: assignmentID is required parameter
    at Object.openAssignment (constellation-core.5de76a5a.js:2:236472)
    at ListView.clickRowInGrid (index.ts:339:35)
    
So, perhaps we should do a screen share to see if that's a problem beyond the scope of this PR and something that will be fixed later.

File: src/components/templates/SingleReferenceReadonly/index.ts (hunk 1 above)
Comment (4manasa) [reply src/components/templates/SingleReferenceReadonly/index.ts]: removed the console and updated the PR

**Comment by 4manasa** [comment]
> See comment re: moving a console.log. Also, when I try to test this, I'm still seeing some errors like this:
> 
> Uncaught (in promise) Error: assignmentID is required parameter at Object.openAssignment (constellation-core.5de76a5a.js:2:236472) at ListView.clickRowInGrid (index.ts:339:35)
> 
> So, perhaps we should do a screen share to see if that's a problem beyond the scope of this PR and something that will be fixed later.

Fixed the above console error and updated the PR

**Review by JEAxero — APPROVED** [review]
Reviewed w/Manasa and looks ok to me.

**Pull request merged**